joomla/joomla-cms
# Hide database error prefix

This PR implements the hide of the SQL Table Prefix from error messages if the webseite is not in debugging mode.

In debugging mode we get the full error message.
#### How to test
- produce a SQL Error message in the frontend
- see that the prefixes get shown
- apply this patch
- see the prefixes are replaced with `#__`
- enable debug for your user
- see the prefixes are back

This implements the good idea by @ryandemmer from here: https://github.com/joomla/joomla-cms/issues/8129 Thanks.
